tests/common.go

```diff
@@ -7062,11 +7064,11 @@ const (
 )
 
 // CreateAzureBucket creates bucket in Azure
-func CreateAzureBucket(bucketName string, objectlock bool, mode Mode, retentionDays int, safeMode bool) {
+func CreateAzureBucket(bucketName string, immutability bool, mode Mode, retentionDays int, safeMode bool) {
 	// From the Azure portal, get your Storage account blob service URL endpoint.
 	_, _, _, _, accountName, accountKey := GetAzureCredsFromEnv()
 	azureRegion := os.Getenv("AZURE_ENDPOINT")
-	if objectlock == true {
+	if immutability == true {
```

Comment:
nit: We can just say `if immutability {`

Reply:
Done